### PR TITLE
Associate te

### PR DESCRIPTION
**Pull request by tarraneh**
Adding FRB190608 to frbs.py

### CODE REVIEW

File: frb/associate/frbassociate.py [hunk 1]
```diff
@@ -14,8 +14,8 @@
 from astropy import wcs as astropy_wcs
 
 from frb import frb
-from frb.associate import bayesian
-from frb.associate import chance
+import bayesian
```
Comment (profxj) [inline frb/associate/frbassociate.py:17]: not sure this will work for others.
Please put it back the way it was.  Thanks!

File: frb/igm.py [hunk 2]
```diff
@@ -0,0 +1,465 @@
+""" Module for IGM calculations
```
Comment (profxj) [inline frb/igm.py:1]: this module is now in `frb/dm`

Please remove it from here.  Thanks!

**Review by profxj — CHANGES_REQUESTED** [review]
2 minor requests.

Thanks!

File: frb/associate/frbassociate.py (hunk 1 above)
Comment (tarraneh) [reply frb/associate/frbassociate.py]: Changed it back!

File: frb/igm.py (hunk 2 above)
Comment (tarraneh) [reply frb/igm.py]: Removed!

**Comment by profxj** [comment]
thx!  merging

**Pull request merged**